### DISCUSSION TITLE
LoRa/Dreambooth training on COLAB - how to get it running?

### DISCUSSION THREAD

**Discussion by krummrey** [d8ahazard/sd_dreambooth_extension#807] [Q&A] [answered]
Hi, 
My machine can barely render images, but training a checkpoint or embedding just doesn't work. So I use google colab for that. Specifically the [notebooks provided by LastBen](https://github.com/TheLastBen/fast-stable-diffusion). Both work well, however, the dreambooth there does not offer LoRa training. And the Automatic1111 notebook does not work with this extension. It either does not show up at all after the one-click installation. Or when I pip install the requirements.txt from this extension gradio breaks:
`ImportError: cannot import name 'queueing' from partially initialized module 'gradio' (most likely due to a circular import) (/usr/local/lib/python3.8/dist-packages/gradio/__init__.py)`

Has anyone gotten this extension to work on a colab installation of Automatic1111?

**Comment by d8ahazard** [comment, marked as answer]
Ask @thelastben to update his Collab after I merge LigherLora to main...

**Reply by krummrey** [reply]
Thanks for your reply, any time frame on when that will happen?
